tests/utils.py

```diff
@@ -12,6 +13,9 @@ def unfix(n):
 def stringToBytes(value):
     return value.ljust(32, '\x00')
 
+def longTo32Bytes(value):
+    return pack(">l", value).rjust(32, '\x00')
```

Comment:
Very nice.  This is even less code than I expected.

Reply:
Thanks, I had seen some examples using `pack()` on StackOverflow, but since they weren't producing the desired number of bytes, I got the idea to use `rjust()` from the stringToBytes() function above.